### DISCUSSION TITLE
Button mapping

### DISCUSSION THREAD

**Discussion by annoyinganime** [OpenI6X/opentx#142] [Ideas]
I was thinking a lot for a couple of days, can't we make controls more intuitive? Jumper t12, for example, has 6 buttons, i6x only has 5, but is the difference that big really? In most cases we don't need dual-axis scrolling, and basic "press OK to select, scroll to needed value, press OK once more to deselect" should be fine. Where am i wrong?

**Comment by ajjjjjjjj** [comment]
Can you show real examples of what do you consider non intuitive and how this could be remapped?

**Reply by jakeeri** [reply]
In my opinion this mapping is intuitive. This is as close as you can get without scrollwheel, SYS and MDL buttons which I am used to with TX-16S.
Especially the use of the right side potentiometer for  selecting pages back and forth and left for selecting values is intuitive and good use for those pots.

**Comment by marianomd** [comment]
For me it's ok like this too. I would add long press to bind button to go back (right page), but not urgent.

**Comment by ajjjjjjjj** [comment]
I also agree that buttons mappings are OK.
There is only one bug in flight modes. I'll add issue for it.

**Comment by annoyinganime** [comment]
The most unintuitive thing is the potentiometer scroll. Well, not "unintuitive", but but extremely awkward to use, because you have to reach all the way up to change one setting, when you can just press OK to select it, change it using UP/DOWN keys, and then press OK to save it

**Reply by ajjjjjjjj** [reply]
You can use UP/DOWN to change focused value, so what is the problem?

**Comment by annoyinganime** [comment]
Seems like I found the control scheme I was trying to write about: [taranis x9d plus](https://imgaz1.staticbg.com/thumb/large/upload/2014/01/SKU15967901.jpg) has 6 buttons, one of which is "PAGE". So, basically, we just need to dump that one, and use rest of our buttons as "menu", "enter", "exit", "+" and "-". And to scroll through pages we just go all the way above to page number, press "enter", and then use + or - to scroll through. Then we don't need to use any scroll wheels, potentiometers, or additional buttons

**Comment by ajjjjjjjj** [comment]
I have merged a change where BIND button now serves as LEFT by long press, while POT2 menu navigation is disabled.
